### ISSUE TITLE
Bug: Fly hijack to terminal does not handle carriage returns properly

### ISSUE THREAD

**Issue by lavjain** [concourse/concourse#1752]
* Concourse version: 2.7 onwards
* Deployment type (BOSH/Docker/binary): BOSH
* Infrastructure/IaaS: AWS
* Browser (if applicable):
* Did this used to work? Yes (worked fine about 6-9 months ago)

I always have to run "reset" command after hijacking into the container.
```
ljain@Lavs-MacBook-Pro pxf_automation (aggr_queries_with_null) $ fly -t hawq-ci hijack -j hdb/pxf_automation_centos6
1: build #124, step: singlecluster, type: get
2: build #130, step: features, type: task, attempt: 1
3: build #134, step: features, type: task, attempt: 2
4: build #139, step: features, type: task, attempt: 1
5: build #139, step: features, type: task, attempt: 2
6: build #139, step: smoke, type: task, attempt: 1
choose a container: 2
[root@55fc3076-1f9b-4bf1-79a6-aeb792860c61 4f944542]# ls -l
                                                           total 0
                                                                  drwxr-xr-x 1 root    root     58 Sep 18 01:45 curl_bin_tarball
                                                                                                                                drwxr-xr-x 1 root    root    128 Oct 11 21:25 hawqrepo
                drwxr-xr-x 1 root    root    100 Oct 11 04:57 hawq_rpm
                                                                      drwxr-xr-x 1 root    root    140 Sep 19 08:12 hdb
                                                                                                                       drwxr-xr-x 1 gpadmin gpadmin 312 Oct 11 22:03 hdfsrepo
       drwxr-xr-x 1 gpadmin gpadmin 522 Oct 11 22:03 pxf_automation
                                                                   [root@55fc3076-1f9b-4bf1-79a6-aeb792860c61 4f944542]#                                                                   [root@55fc3076-1f9b-4bf1-79a6-aeb792860c61 4f944542]# reset

[root@55fc3076-1f9b-4bf1-79a6-aeb792860c61 4f944542]# ls -l
total 0
drwxr-xr-x 1 root    root     58 Sep 18 01:45 curl_bin_tarball
drwxr-xr-x 1 root    root    128 Oct 11 21:25 hawqrepo
drwxr-xr-x 1 root    root    100 Oct 11 04:57 hawq_rpm
drwxr-xr-x 1 root    root    140 Sep 19 08:12 hdb
drwxr-xr-x 1 gpadmin gpadmin 312 Oct 11 22:03 hdfsrepo
drwxr-xr-x 1 gpadmin gpadmin 522 Oct 11 22:03 pxf_automation
[root@55fc3076-1f9b-4bf1-79a6-aeb792860c61 4f944542]#
```

**Comment by lavjain** [comment]
I didn't report this bug earlier because it looked like a temporary glitch that I was expecting to get fixed in the newer versions but the issue is still there. If fixed, it saves time for many developers by not having to type reset everytime they hijack into a container.

**Comment by jtarchie** [comment]
@lavjain, are you able to provide a reproducible example?

**Comment by lavjain** [comment]
I just tried another concourse instance (running on Google Cloud) with concourse ver 3.4.1 and surprisingly it does not happen. It could be something that was fixed between 2.7 and 3.4 or AWS specific. Let me find out more...

**Comment by lavjain** [comment]
Closing the issue for now, since I cannot reproduce it with Concourse ver 3.4.1
